package.json

```diff
@@ -18,28 +18,50 @@
     "scripts": {
         "dev": "rollup -m -c",
         "build": "rimraf build && rollup -m -c --environment NODE_ENV:production",
-        "test": "mocha"
+        "postbuild": "echo '{\"type\": \"commonjs\"}'> build/cjs/package.json",
+        "test": "npm run test:node && npm run test:browser",
+        "test:node": "NODE_OPTIONS='--experimental-vm-modules --no-warnings' jest --coverage",
+        "test:browser": "NODE_OPTIONS='--experimental-vm-modules --no-warnings' jest --coverage --testEnvironment=jsdom"
     },
     "author": "Jose F. Romaniello <jfromaniello@gmail.com> (http://joseoncode.com)",
```

Comment:
```suggestion
    "author": "Jose F. Romaniello <jfromaniello@gmail.com> (https://joseoncode.com)",
```

Reply:
This site doesnt even exist so I will remove it.